crawler/emitter.py

```
@@ -241,6 +240,61 @@ def _publish_to_stdout(self):
                     print line.strip()
                     sys.stdout.flush()
 
+    def _publish_to_logstash(self, url, max_emit_retries=5):
```

Those unsupported cases should raise a NotImplementedError exception